README.md

```diff
@@ -58,3 +95,17 @@ npm install --save alpaca-trade-api
 
 ### Calendar API
 * `getCalendar(start, end)`: Calls `GET /calendar` and returns the market calendar.
+
+### Websockets
+* `let websocket = new alpaca.websockets.client();`: Create a websocket client instance.
+* `websocket.connect()`: Connect to the alpaca server using websocket.
+* `websocket.subscribe(channels)`: Subscribe to the alpaca server and possibly Polygon server
+    Possible channels: 'trade_updates', 'account_updates', 'T.*', 'Q.*', 'A.*', AM.*'
+        The first two are for the alpaca server, the rest are for the Polygon server.
+        For more information, please contact the relevant pages.
+* `websocket.onOrderUpdate(function(data))`: Register callback function for the channel 'trade_updates'.
+* `websocket.onAccountUpdate(function(data))`: Register callback function for the channel 'account_updates'.
+* `websocket.onStockTrades(function(data))`: Register callback function for the channel 'T.*'.
+* `websocket.onStockQuotes(function(data))`: Register callback function for the channel 'Q.*'.
+* `websocket.onStockAggSec(function(data))`: Register callback function for the channel 'A.*'.
+* `websocket.onStockAggMin(function(data))`: Register callback function for the channel 'AM.*'.
```

Comment:
Should document the fact that the websocket is a node EventEmitter, and these are just aliases for the `on` method, bound to specific events.